tests/test_pipenv.py

```diff
@@ -934,6 +934,25 @@ def test_complex_lock_with_vcs_deps(self, pip_src_dir, pypi):
             assert 'python_dateutil' not in lock['default']
             assert 'python_dateutil' not in lock['develop']
 
+    @pytest.mark.lock
+    @pytest.mark.requirements
+    def test_lock_with_prereleases(self, pypi):
+
+        with PipenvInstance(pypi=pypi) as p:
+            with open(p.pipfile_path, 'w') as f:
+                contents = """
```

Comment:
what do you think about using dedent to keep a nice pythonic flow?

Reply:
In fact TOML is insensitive of indents, I think we can just add them to make it prettier